packages/api/src/middleware/validators.js

```diff
@@ -1,5 +1,5 @@
 // import { schemaWalk } from '@cloudflare/json-schema-walker'
-import validators from "../schema/validators";
+import validators from "../../dist/schema/validators";
```

Comment:
Won't this break the builds before a build is ever made? Chicken&egg

Reply:
Nah, the babel compilation doesn't care that the module doesn't exist, it literally just transpiles the source. The schema generation is [its own thing](https://github.com/livepeer/studio/blob/357a24e9a39f5c441cc89270a4415cc857138c93/packages/api/src/compile-schemas.js) that only ever puts files in the `dist` directory. Previously we'd only ever execute from `dist` but now the `esbuild` compiles directly out of `src`, so we need to point to their actual location.

Reply:
Wait no you're totally right it writes it to the `src` directory, fixing